osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -92,11 +92,21 @@ private void load()
                     }
                 }
             });
-
-            loadNewPlayer();
         }
 
-        private void playerLoaded(Player player) => info.Loading = false;
+        private void playerLoaded(Player player)
+        {
+            if (IsDisposed)
```

Comment:
this should never be called. the callback is invoked on this.scheduler`, which would not be running if `IsDisposed` is true. also if you're going to add super-weird disposal logic like this, please break out of a completely unrelated PR.